sanity-check/main.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+  Copyright (C) 2022,  icgc-argo
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  Authors:
+    Edmund Su
+"""
+
+from email import contentmanager
+import os
+import sys
+import argparse
+import subprocess
+import json
+import csv
+import requests
+
+
+def main():
+    """
+    Python implementation of tool: sanity-check
+
+    This is auto-generated Python code, please update as needed!
+    """
+
+    parser = argparse.ArgumentParser(description='Tool: sanity-check')
+    parser.add_argument('-t', '--api_token', dest='api_token', type=str,
+                        help='Platform UUID token', required=True)
+    parser.add_argument('-c', '--clinical_url', dest='clinical_url', type=str,
+                        help='Clinical SONG API interaction URL', required=True)
+    parser.add_argument('-s', '--submission_song_url', dest='submission_song_url',type=str,
+                        help='Submission SONG API interaction URL', required=True)
+    parser.add_argument('-x', '--experiment_info_tsv', dest='experiment_info_tsv', type=str,required=True,
+                        help='TSV contain experiment info and submitter IDs')
+    parser.add_argument('-f', '--force', action='store_true',help='JSON file containing metadata payload')
+    args = parser.parse_args()
+
+    if args.experiment_info_tsv:
+        metadata=load_tsv(args.experiment_info_tsv)
+        
+        clinical_metadata=get_clinical(
+            metadata,
+            args.clinical_url,
+            args.api_token
+        )
+        
+        final_metadata=compile_metadata(
+            metadata,
+            clinical_metadata
+        )
+        if not args.force:
+            check_study_exists(
```

Comment:
`check_study_exists` should be always checked.

Reply:
Updated:
https://github.com/icgc-argo/argo-data-submission/blob/2a6ee2ba6bf1716bd58ed3d3e008bc08594d0e9c/sanity-check/main.py#L66-L70